src/jobs.liquid

```diff
@@ -0,0 +1,57 @@
+---
+layout: content.liquid
+---
+
+<section class="container pt-md-4 pb-md-2" id="job-board">
+  <h1 class="text-center">Job Board</h1>
+  <div class="jobs-container">
+  {% if jobs.length > 0 %}
+    {% for job in jobs %}
+      <div class="col-12 d-flex mb-lg-0 mb-4 mt-4 p-2">
+        <div class="jobcard card shadow p-4 rounded-md">
+          <li class="d-flex flex-column">
```

Comment:
I don't think this and the rest of the `<li>`s below should be `<li>`. They can just be simple `div`I think.